src/people/widgetViews/organization/EditOrgModal.tsx

```diff
@@ -215,12 +216,15 @@ const EditOrgModal = (props: EditOrgModalProps) => {
       const newOrg = {
         id: org.id,
         uuid: org.uuid,
-        name: body.name || org.name,
+        name: normalizeInput(body.name) || org.name,
         owner_pubkey: org.owner_pubkey,
         img: img || org.img,
-        description: body.description !== undefined ? body.description : org?.description || null,
-        github: body.github !== undefined ? body.github : org?.github || null,
-        website: body.website !== undefined ? body.website : org?.website || null,
+        description:
+          normalizeTextValue(body.description) !== undefined
+            ? body.description
+            : org?.description || null,
+        github: normalizeUrl(body.github) !== undefined ? body.github : org?.github || null,
+        website: normalizeUrl(body.website) !== undefined ? body.website : org?.website || null,
```

Comment:
we are just checking if the normalized value doesn't equal undefined here, but it can still be malformed and then we'd pass the malformed value

wouldn't something like this make more sense?
`github: normalizeUrl(body.github) !== undefined ? normalizeUrl(body.github) : normalizeUrl(org?.github) || null,`